public/server/db.json

```diff
@@ -72,7 +72,13 @@
     },
     
     "lol":{
-        "definition": "Laugh Out Loud"
+        "definition": "Laugh Out Loud",
+        "definition": "Lots Of Love",
```

Comment:
Minor change, instead of having 6 `definition` key value pairs, please keep one (I recommend Laugh Out Loud because it is most popular) and wrap the rest of them in one `alternatives` comma separated key value pair.  Because of the way it's implemented now, we are unable to see all the alternatives you worked so hard on compiling:
![Screen Shot 2022-10-04 at 7 07 57 PM](https://user-images.githubusercontent.com/9134674/193952461-56b9709c-97a5-489b-8b40-020b2cabc28b.jpg)

You can look at [here](https://github.com/Njong392/Abbreve/blob/25f20612768f1407107e2957a9df2a1b378d71b9/public/server/db.json#L8) for a good example of how it should be implemented.

Reply:
Thanks for noticing, I'll make changes here